.percy.js

```diff
@@ -1,11 +1,6 @@
 const SNAPSHOTS = [
-  '/index.html', // Home
-  '/docs/index.html', // Docs
-  '/docs/native-client/index.html', // Docs landing page
-  '/docs/extensions/mv3/index.html', // Docs page
-  '/docs/extensions/reference/action/index.html', // API reference page
-  '/docs/handbook/test-post/index.html', // Test blog post
-  '/docs/handbook/components/index.html', // Handbook components
+  '/en/page-types/author-individual/index.html',
+  '/en/test-post/index.html',
```

Comment:
Would this make sense to move test-post to the page-types dir?

Reply:
Yeah that makes total sense. I've actually done so in the second [PR](https://github.com/GoogleChrome/developer.chrome.com/pull/4450/files) which will be ready to go pretty much straight away once this one is merged in.